tutor/commands/k8s.py

```diff
@@ -339,17 +342,34 @@ def delete(context: K8sContext, yes: bool) -> None:
     )
 
 
+@jobs.do_group
+@click.pass_obj
+def do(context: K8sContext) -> None:
+    """
+    Run custom jobs in the right containers.
```

Comment:
The rest of the command helptexts are written in a singular voice (e.g. `run  --  Run a command in a new container`) so I suggest doing the same here.
```suggestion
    Run a custom job in the right containers.
```